test/e2e/framework/framework.go

```diff
@@ -248,6 +248,9 @@ func (f *Framework) BeforeEach() {
 			ginkgo.By("Waiting for a default service account to be provisioned in namespace")
 			err = WaitForDefaultServiceAccountInNamespace(f.ClientSet, namespace.Name)
 			ExpectNoError(err)
+			ginkgo.By("Waiting for kube-root-ca.crt to be provisioned in namespace")
+			err = WaitForKubeRootCAInNamespace(f.ClientSet, namespace.Name)
+			ExpectNoError(err)
```

Comment:
Pet peeve of mine: the `err` is a pretty uninformative "timed out waiting for the condition", so without an additional message, a failure is just:
```
Apr 21 13:00:31.563: FAIL: Unexpected error:
    <*errors.errorString | 0xc00042cd80>: {
        s: "timed out waiting for the condition",
    }
    timed out waiting for the condition
occurred
```

I know because I am hitting this in one of my downstream test scenarios... :sweat_smile: 

Please always add an explanation when calling ExpectNoError.